src/utils/activation_functions.jl

```diff
@@ -222,3 +223,39 @@ function GaLUgrad(Δy::AbstractArray{Float32, 5}, x::AbstractArray{Float32, 5})
     Δx[:, :, :, k+1:end, :] = SigmoidGrad(Δy, nothing; x=x2) .* x1
     return Δx
 end
+
+###############################################################################
+# Soft-clamped exponential function
+
+"""
+    y = ExpClamp(x)
+
+ Soft-clamped exponential function.
+
+ See also: [`ExpClampGrad`](@ref)
+"""
+function ExpClamp(x::Array{Float32}; clamp::Float32=2f0)
+    return exp.(clamp * 0.636f0 * atan.(x))
```

Comment:
"0.636f0" that's an interesting number.  Can't it be just absorbed in `clamp` to avoid surprises?

Reply:
That weirdly specific choice is calculated by 2/pi. See here: https://github.com/VLL-HD/FrEIA#documentation:

> The RNVP and GLOW coupling blocks have an additional hyperparameter clamp. This is becuase, instead of the exponential function exp(s), we use exp( 2*c/pi * atan(x)) in the coupling blocks (clamp-parameter c). This leads to much more stable training and enables larger learning rates. Effecively, the mutliplication component of the coupling block is limited between exp(c) and 1/exp(c). The Jacobian determinant is thereby limited between ±D*c (dimensionaltiy of data D). In general, clamp = 2.0 is a good place to start:

transf1 = Node([in1.out0], GLOWCouplingBlock,
               {'subnet_constructor':fc_constr, 'clamp':2.0},
               name='Transform T_1')  >

I would suggest to keep that constant like it is now, but add some more documentation on the function itself